client/src/InfoBase/App.js

```diff
@@ -118,15 +118,11 @@ export class App extends React.Component {
   }
 
   state = {
-    showNotification: false,
-    message: "",
+    show_redirected_msg: sessionStorage.getItem("redirected_msg"),
```

Comment:
Ooh, instead of `sessionStorage`, this could finally be a good reason to make use of the app-wide provider. Would deal with that old `No reducer provided for key "app"` error message in the process, haha.

Reply:
Well, with how it is now, it'll need to keep its variables on reload, and I don't think redux storage will keep that! (Although if we do this at Infographic level, then maybe we can use react router dom and it might have some redirect stuff that doesn't reload? I'd have to try that out)

Reply:
Ah, hmm, yeah. Just need to catch the error inside the route rather than let it hit the error boundary and junk everything up 👍